types/react-redux/react-redux-tests.tsx

```diff
@@ -250,7 +250,7 @@ namespace MapStateAndOptions {
     interface State { state: string; }
     interface OwnProps { foo: string }
     interface StateProps { bar: number }
-    interface DispatchProps { dispatch: Dispatch }
+    interface DispatchProps { dispatch: Dispatch<State> }
```

Comment:
isn't the `Dispatch` generic typed as `Dispatch<Action>` now?

Reply:
My mistake, my `types/react-redux/node_modules` was outdated. Fixed now!